src/main/resources/templates/datewarehouse/etl-jobs-config.json

```diff
@@ -94,30 +276,107 @@
 		"sourcePath" : "log.sagebase.org/fileSnapshots/",
 		"tableDescriptor": {
 			"name": "filesnapshots",
-			"columns": {
-				"change_type": "string",
-				"change_timestamp": "timestamp",
-				"change_user_id": "bigint",
-				"snapshot_timestamp": "timestamp",
-				"id": "bigint",
-				"created_by": "bigint",
-				"created_on": "timestamp",
-				"modified_on": "timestamp",
-				"concrete_type": "string",
-				"content_md5": "string",
-				"content_type": "string",
-				"file_name": "string",
-				"storage_location_id": "bigint",
-				"content_size": "bigint",
-				"bucket": "string",
-				"key": "string",
-				"preview_id": "bigint",
-				"is_preview": "boolean",
-				"status": "string"
-			},
-			"partitionKeys": {
-				"snapshot_date": "date"
+			"description" : "This table contain snapshots of file handles. Snapshots are taken when file handles are created, updated or deleted. Note: Snapshots are also taken periodically and independently of the changes. The snapshot_timestamp records when the snapshot was taken.",
+			"columns":[{
+				"name": "change_type",
+				"type" : "string",
+				"comment" : "The type of changed that occurred on file handle, e.g., CREATE, UPDATE, DELETE."
+			},
+			{
+				"name": "change_timestamp",
+				"type" : "timestamp",
+				"comment" : "The time when change(created/updated/deleted) on file is pushed to the queue for snapshotting."
+			},
+			{
+				"name": "change_user_id",
+				"type" : "bigint",
+				"comment" : "who made the change to the file."
+			},
+			{
+				"name": "snapshot_timestamp",
+				"type" : "timestamp",
+				"comment" : "The time when the snapshot was taken (It is usually after the change happened)."
+			},
+			{
+				"name": "id",
+				"type" : "bigint",
+				"comment" : "The identifier of the file handle."
+			},
+			{
+				"name": "created_by",
+				"type" : "bigint",
+				"comment" : "The identifier of the user who created the file handle."
+			},
+			{
+				"name": "created_on",
+				"type" : "timestamp",
+				"comment" : "The creation timestamp of the file handle."
+			},
+			{
+				"name": "modified_on",
+				"type" : "timestamp",
+				"comment" : "The most recent change timestamp of the file handle."
+			},
+			{
+				"name": "concrete_type",
+				"type" : "string",
+				"comment" : "The type of file handle, e.g., s3, google cloud, external file handle."
```

Comment:
Is there a link to the supported concrete_types?

Reply:
There is no list but I have added allowed file handles. The type of file handle. Allowed file handles are : S3FileHandle, ProxyFileHandle, ExternalFileHandle, ExternalObjectStoreFileHandle, GoogleCloudFileHandle.